main.c

```diff
@@ -291,6 +291,8 @@ void drawEnemy(ENEMY *block) {
 void setTextTexture(TEXT_TEXTURE *txtText, TTF_Font *font, char *text) {
   // Render text surface
   SDL_Surface *textSurface = TTF_RenderText_Solid(font, text, textColor);
+  // Temp hack so it boots on Xbox
+  if(txtText->txtText != NULL)
```

Comment:
This shouldn't be an issue; [via SDL wiki](https://wiki.libsdl.org/SDL_DestroyTexture):

> Passing NULL or an otherwise invalid texture will set the SDL error message to "Invalid texture". 

Did this really cause problems? We never touched the SDL renderer, so it shouldn't do anything bad

Reply:
in SDL2/src/render/SDL_render.c:

```c
#define CHECK_TEXTURE_MAGIC(texture, retval) \
    SDL_assert(texture && texture->magic == &texture_magic); \
    if (!texture || texture->magic != &texture_magic) { \
        SDL_SetError("Invalid texture"); \
        return retval; \
    }
```

I assume your `SDL_assert` is different from your development platforms.

You should fix this as a bug in the basegame (move commit to before Xbox port).
Remember that if you compare to `NULL`, that you'll also have to make sure that these pointers are initialized to `NULL` when they are being initialized / allocated.